test/test_helper.rb

```diff
@@ -1,5 +1,3 @@
-lib_dir = File.dirname(__FILE__) + '/../lib'
-
+require 'bundler/setup'
```

Comment:
> I don't think this is necessary.
> What if bundler is added as a development dependency to the gemspec?

Do not worry, if `require "... debug_inspector" is gone in `test/test_helper.rb `, I just patch to add the line in the project.

But your suggestion "Or, something like:" is better for me.